README.md

```diff
@@ -37,6 +37,7 @@ Tigase's Docker package for the Tigase XMPP Server.
 
 ## Simple tags
 -   [`nightly` (*nightly/Dockerfile*)](nightly/Dockerfile)
+-   [`nightly-jre16` (*nightly/Dockerfile*)](nightly/Dockerfile)
```

Comment:
Shouldn't the path here points to `nightly/jre-16/Dockerfile`?

Reply:
Yes, you are correct, I'll fix it.